packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/extension.tsp

```diff
@@ -26,19 +28,20 @@ model ExtensionsResourceProperties {
 }
 
 @armResourceOperations
-interface ExtensionsResources {
+@clientName("ExtensionsResources")
```

Comment:
This is new test, guess we don't need `@clientName`?

Reply:
Rollback by [commit#7331206](https://github.com/Azure/cadl-ranch/pull/701/commits/7331206a2fa5cd52817a57f888095b9ea497821b) and [commit#69a4d5b](https://github.com/Azure/cadl-ranch/pull/701/commits/69a4d5b2fec7eaeccfe4dd706865675d501283c9)